<!-- Map Monitor (without PythonAPI) launch configuration -->
<launch>
    <!-- Arguments set by terminal 
    <arg name='map_name' default='Town03'/> -->

    <!-- Parameters of the layer 
    <group ns="/t4ac/mapping">
        <param name="map_path" type="string" value="/workspace/team_code/catkin_ws/src/t4ac_mapping_layer/maps/xodr/"/>
        <param name="map_name" type="string" value="$(arg map_name)" />> 
    </group>  -->

    <!-- Nodes launched -->
    <node pkg="t4ac_utils_ros" name="t4ac_viewer_node" type="t4ac_viewer.py" output="screen"/>
</launch>